src/sys/socket/sockopt.rs

```diff
@@ -368,7 +368,7 @@ impl<'a> Set<'a, usize> for SetUsize {
     }
 
     unsafe fn ffi_ptr(&self) -> *const c_void {
```

Comment:
`unwrap` and `blank` should probably stay unsafe: The caller must uphold safety for those, not the callee or implementor.